dist/.log.env.md

```diff
@@ -0,0 +1,90 @@
+### Set up notifications
```

Comment:
The organization of this file doesn't make a lot of sense to me. Slack & Email configuration are log handlers and should go under there (that would mean modifying some of the language as well). I think we should re-order to the following:
```
# Logging Handlers
## Slack Configuration
## Email Configuration
# Log Levels
```

Reply:
Also, on devdocs we have two pages about the logging configuration.
https://devdocs.magento.com/cloud/env/log-handlers.html
https://devdocs.magento.com/cloud/env/setup-notifications.html

@meker12 could you please take a look at these pages and this file, we need to combine it somehow better.